build.sh

```diff
@@ -13,4 +13,4 @@ while [[ -h $source ]]; do
 done
 
 scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
-"$scriptroot/eng/common/build.sh" --pack --build --restore $@
```

Comment:
It's odd that --pack is specified in the default build.sh, we don't do this elsewhere. Is there a reason for this in this repo?

Reply:
not sure, but we'd probably need to update the azure-pipelines yaml to add -pack if we remove it.

Reply:
azure-pipelines uses eng/common/cibuild.sh which passes --pack so it's not necessary to update anything. I took a look and it seems nothing else in the repo references or uses build.sh directly.

Reply:
Alright, update it then.

Reply:
Update what?

Reply:
Oh, you already removed pack... 😄